docs/api-reference/io-columns-multiclass-classification.md

```diff
@@ -0,0 +1,7 @@
+### Input and Output Columns
+The input label column data must be key typed. This trainer outputs the following columns:
+
+| Output Column Name | Column Type | Description|
+| -- | -- | -- |
+| `Score` | array of <xref:System.Single> | The scores of all classes. Higher value means higher probability to fall into the associated class. If the i-th element has the lagest value, the predicted label index would be i. Note that i is zero-based index. |
```

Comment:
>lagest [](start = 172, length = 6)

typo: largest

please use spell checker. we won't catch all the typos manually.
[https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellChecker](https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellChecker)